gladis/src/lib.rs

```diff
@@ -142,3 +170,4 @@ pub trait Gladis {
 #[cfg(feature = "derive")]
 #[doc(hidden)]
 pub use gladis_proc_macro::Gladis;
+use std::{error::Error, fmt::Display};
```

Comment:
I am confused why this is at the end of the file and not at the top, just before `use gtk;`. Can you move it up?

Reply:
Sure, this is just where rust-analyzer choose to insert it. There is no `use gtk;` in this file so I just moved it to the standard location bellow the module documentation, is that ok?

Reply:
You did well, I mixed with another file.